tests/data/helloworld.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Hello world Lambda function for mu testing.
+"""
+
+
+def main(event, context):
+    for rec in event['Records']:
```

Comment:
Went with `json.dump` in fd9ccf9d58b0f6319f7fc6abaf5bcea08d850fdc, for more natural parsing on the flip side (avoiding `eval`).